tests/testthat/setup.R

```diff
@@ -0,0 +1,9 @@
+library(dplyr)
+
+## listings var labels don't get mucked up by topleft machinery #262
+anl <- formatters::ex_adsl
```

Comment:
formatters is a formal dependency, not a suggests so we don't need `::` here

Reply:
This is related to https://github.com/insightsengineering/rlistings/issues/49 and I agree in this case, as it is a `Depends:`. Still, I think it helps clarify a lot to have explicit calls to `formatters` as described in #49. It is possible that external devs or users could save a lot of time by having clear dependencies

Reply:
for data, I removed the `::`